tests/test_services.py

```diff
@@ -215,7 +215,7 @@ async def test_clear_code(hass, lock_data, sent_messages, caplog):
         await hass.services.async_call(DOMAIN, SERVICE_CLEAR_CODE, servicedata)
         await hass.async_block_till_done()
         assert (
```

Comment:
Maybe add a comment here stating that we're setting the code before attempting to clear it when using `zwave` ?
Just for future references.